compiler/rustc_resolve/src/late/diagnostics.rs

```diff
@@ -1151,7 +1151,7 @@ impl<'a: 'ast, 'ast, 'tcx> LateResolutionVisitor<'a, '_, 'ast, 'tcx> {
 
     fn suggest_changing_type_to_const_param(
         &mut self,
-        err: &mut Diagnostic,
+        err: &mut DiagnosticBuilder<'_>,
```

Comment:
what does this have to do with anything?

Reply:
Yeah, accidental inclusion from some follow-up changes, I will revert it.